modules/fts/pages/fts-secure-fts-queries.adoc

```diff
@@ -0,0 +1,44 @@
+= Secure Queries Using SSL
+
+== *Secure FTS queries using SSL*
+For querying, the user has to use https and the SSL port (18094), for example,
+----
+curl -XPOST ... https://<ip>:18094/api/index/{indexName}/query ...
+----
+
+The user needs to provide the username, password (-u).
+ 
+For information on using cURL in N1QL, see xref:https://blog.couchbase.com/developer-release-curl-n1ql/[blog]
+
+The curl function takes up to three parameters:
+1. The first parameter is the HTTP verb used. So far, we support “GET” and “POST.”
+2. The second parameter is the endpoint URL
+3. The third parameter represents different cURL options.
+
+== *Security Options*
+
+.Security Options
+[#optional-id,cols="1,1,2",options="header"]    
+|===
+
+|Option
+|Description
+|Value 
+
+|User
+|Server user and password
+|USERNAME[:PASSWORD
+
+|Basic
+|Use HTTP Basic Authentication
+|-
+
+|Insecure 
+|Allow connections to SSL sites without certs (H)
+|-
+
+|Anyauth
+|curl to figure out authentication method by itself, and use the most secure one (In our case this will be basic only for now. Eventually we will support digest)
```

Comment:
curl to figure out authentication method by itself, and use the most secure one. Currently, only basic auth is supported.

Reply:
Updated